examples/cg.py

```diff
@@ -271,8 +276,43 @@ def run_cg(
         dest="conv_threshold",
         help="convergence check threshold",
     )
+    parser.add_argument(
```

Comment:
Add `choices=["legate", "numpy", "cupy"]` to the `--package` option in all files, for better error checking and documentation.

Reply:
Fixed. Thanks.